addons/yippee/userscript.js

```diff
@@ -0,0 +1,137 @@
+export default async function ({ addon, msg, console }) {
+
+//Change font family -- fontSelector in the settings --
+
+var allTextElements = document.querySelectorAll('div, p, label, text, span, a, button, h1');
+
+//The cambiaTexto function is called at the end of the document, in a 'mouseover' eventListener, along with the other functions.
+function cambiaTexto() {
+const fontSelector = addon.settings.get('fontSelector');
+
+		for (var i = 0; i < allTextElements.length; i++) {
+			if (fontSelector == "arial") {
+				allTextElements[i].style.fontFamily = "Arial, Helvetica, sans-serif";
+			}
+			else if (fontSelector == "verdana") {
+				allTextElements[i].style.fontFamily = "Verdana, Tahoma, sans-serif";
+			}
+			else if (fontSelector == "comicSans") {
+				allTextElements[i].style.fontFamily = "'Brush Script MT', Cursive";
+			}
+		}
+}
+
+//Change font size -- fontSizer in the settings --
+
+//This function will also be called at the end.
+function cambiaTamaño() {
+	for (var j = 0; j < allTextElements.length; j++) {
+			allTextElements[j].style.fontSize = addon.settings.get('fontSizer')+'px';
+		}
+}
+
+//Change text to Pictograms -- texToPic in the settings --
+
+//This function will also be called at the end.
+function cambiaPictos() {
+	if (addon.settings.get('texToPic') == true) {
```

Comment:
Note that the "Block palette category icons" addon exists, which already does a similar thing (though it only adds icons, not replaces the text with them).